tutorials/intro/plot_40_sensor_locations.py

```diff
@@ -96,6 +96,88 @@
 fig.gca().view_init(azim=70, elev=15)
 ten_twenty_montage.plot(kind='topomap', show_names=False)
 
+###############################################################################
+# .. _control-chan-projection:
+#
+# Controlling channel projection (MNE vs EEGLAB)
+# ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+#
+# Channel positions in 2d space are obtained by projecting their actual 3d
+# positions onto a sphere. Because ``'standard_1020'`` montage contains
+# realistic, not spherical, channel positions, we will use a different montage
+# to demonstrate controlling how channels are projected to 2d space.
+
+biosemi_montage = mne.channels.make_standard_montage('biosemi64')
+biosemi_montage.plot(show_names=False)
+
+###############################################################################
+# By default a sphere  with an origin in ``(0, 0, 0)`` x, y, z coordinates and
+# radius of ``0.095`` meters (9.5 cm) is used. You can use a different sphere
+# radius by passing a single value to ``sphere`` argument in any function that
+# plots channels in 2d (like :meth:`~mne.channels.DigMontage.plot`that we use
```

Comment:
/home/circleci/project/doc/auto_tutorials/intro/plot_40_sensor_locations.rst:250: WARNING: py:meth reference target not found: mne.channels.DigMontage.plot`that we use

Reply:
```suggestion
# plots channels in 2d (like :meth:`~mne.channels.DigMontage.plot` that we use
```

Reply:
You mean the `~` is unnecessary? I thought I copied it form elsewhere in this tutorial...

Reply:
No the `~` is fine, the missing space is the problem

Reply:
Right, I didn't notice that. :D